modules/ptp-configuring-linuxptp-services-as-boundary-clock-dual-nic.adoc

```diff
@@ -0,0 +1,102 @@
+// Module included in the following assemblies:
+//
+// * networking/using-ptp.adoc
+
+:_content-type: PROCEDURE
+[id="ptp-configuring-linuxptp-services-as-bc-for-dual-nic_{context}"]
+= Configuring linuxptp services as boundary clocks for dual NIC hardware
+
+You can configure the `linuxptp` services (`ptp4l`, `phc2sys`) as boundary clocks for dual NIC hardware by creating a `PtpConfig` custom resource (CR) object for each NIC.
+
+Dual NIC hardware allows you to connect each NIC to the same upstream leader clock with separate `ptp4l` instances for each NIC feeding the downstream clocks.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+
+* Log in as a user with `cluster-admin` privileges.
+
+* Install the PTP Operator.
+
+.Procedure
+
+. Create two separate `PtpConfig` CRs, one for each NIC, using the reference CR in "Configuring linuxptp services as a boundary clock" as the basis for each CR. For example:
+
+.. Create `boundary-clock-ptp-config-nic1.yaml`, specifying values for `phc2sysOpts`:
++
+[source,yaml]
+----
+apiVersion: ptp.openshift.io/v1
+kind: PtpConfig
+metadata:
+  name: boundary-clock-ptp-config-nic1
+  namespace: openshift-ptp
+spec:
+  profile:
+  - name: "profile1"
+    ptp4lOpts: "-2 --summary_interval -4"
+    ptp4lConf: | <1>
+      [ens5f1]
+      masterOnly 1
+      [ens5f0]
+      masterOnly 0
+    ...
+    phc2sysOpts: "-a -r -m -n 24 -N 8 -R 16"
+----
+<1> Specify the required interfaces to start `ptp4l` as a boundary clock. For example, `ens5f0` synchronizes from a grandmaster clock and `ens5f1` synchronizes connected devices.
+
+.. Create `boundary-clock-ptp-config-nic2.yaml`, removing the `phc2sysOpts` field altogether to disable the `phc2sys` service for the second NIC:
++
+[source,yaml]
+----
+apiVersion: ptp.openshift.io/v1
+kind: PtpConfig
+metadata:
+  name: boundary-clock-ptp-config-nic2
+  namespace: openshift-ptp
+spec:
+  profile:
+  - name: "profile2"
+    ptp4lOpts: "-2 --summary_interval -4"
+    ptp4lConf: | <1>
+      [ens7f1]
+      masterOnly 1
+      [ens7f0]
+      masterOnly 0
+...
+----
+<1> Specify the required interfaces to start `ptp4l` as a boundary clock on the second NIC.
++
+[NOTE]
+====
+You must completely remove the `phc2sysOpts` field from the second `PtpConfig` CR to disable the `phc2sys` service on the second NIC.
+====
+
+. Create the CRs by running the following commands:
++
```

Comment:
You could add a substep for each of these commands, so that the reader is clear about what each of them do. This will also prevent two command blocks from being adjacent without a descriptive step between them.